cmd/gateway-main.go

```diff
@@ -171,6 +188,9 @@ func gatewayMain(ctx *cli.Context) {
 	// Fetch access and secret key from env.
 	accessKey, secretKey := mustGetGatewayCredsFromEnv()
 
+	// Fetch browser env setting
+	mustGetBrowserSettingFromEnv()
```

Comment:
Rather than `mustGet` it must be `mustSet` since Get is supposed to return, Set is supposed to set / save.